nutrinova-api/NutrinovaApi/Controllers/RecipeController.cs

```diff
@@ -152,44 +153,28 @@ public ActionResult<IEnumerable<NutrientSummary>> SummarizeRecipeNutrients(IEnum
   public async Task<ActionResult<RecipeResponseModel>> GetRecipe(Guid id)
   {
     var recipe = await context.RecipePlans
-      .Include(r => r.RecipeFoods)
```

Comment:
Where you able to see recipes details? 

When I pulled the branch and went to a recipe I got errors.

Reply:
Its working for me, does it say what error?

Reply:
Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware[1]
      An unhandled exception has occurred while executing the request.
      System.ArgumentException: Units must be of the same type Solid != Quantity
         at NutrinovaData.Features.Units.UnitConverter.Convert(Decimal value, Unit fromUnit, Unit toUnit) in /app/nutrinova-api/NutrinovaData/Features/Units/UnitConverter.cs:line 51
         at NutrinovaData.Features.Recipes.RecipeFoodTotaler.GetNutrientSummaries(List`1 recipeFoods) in /app/nutrinova-api/NutrinovaData/Features/Recipes/RecipeFoodTotaler.cs:line 24
         at NutrinovaApi.Controllers.RecipeController.GetRecipe(Guid id) in /app/nutrinova-api/NutrinovaApi/Controllers/RecipeController.cs:line 175

Reply:
I don't know if I need to blow away my database but idk?

Reply:
You have some bad data somehow. Somehow a food was added with a unit that doesnt match the serving size unit of the food, so it cant convert it

Reply:
I guess you can get around the constraint in adding a food to the recipe, because it doesnt constrain the units to  a category until you select a food, maybe we should make it a stepper so you have to select a food first before the unit

Reply:
Gotcha

Reply:
Got rid of the database and made new recipe and it worked